estate/models/estate_property.py

```diff
@@ -0,0 +1,23 @@
+from odoo import fields, models
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(string="Name", required=True)
+    description = fields.Text(string="Description")
+    postcode = fields.Char(string="Postcode")
+    date_availability = fields.Date(string="Availability Date")
+    expected_price = fields.Float(string="Expected Price", required=True)
+    selling_price = fields.Float(string="Selling Price")
+    bedrooms = fields.Integer(string="Bedrooms")
+    living_area = fields.Integer(string="Living Area (sqm)")
+    facades = fields.Integer(string="Facades")
+    garage = fields.Boolean(string="Garage Available")
+    garden = fields.Boolean(string="Has Garden")
+    garden_area = fields.Integer(string="Garden Area (sqm)")
```

Comment:
you set field labels :heart: 
note that for non relational fields (and non selection fields), you can basically do
`name = fields.Char("Name", required=True)`
as string is the first parameter, so you don't have to specify it.